.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Add plugin
+about: Request to add your plugin to the Marketplace
+title: "Add $REPOSITORY_NAME to Marketplace"
+labels: Plugin/New
+assignees: hanzei
+
+---
+<!--
+Thank you very for submitting your Plugin! It will go through a review process to make sure it follow the quality standard  of the Marketplace. This process might take a couple of weeks bepending on how many changes are needed.
+Read https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/ before submitting your plugin.
+-->
+
+#### Summary
+<!--
+A brief description what your plugin does.
+-->
+
+#### Review Commit
+<!--
+Please link the commit or release that should be used for review.
```

Comment:
I realize this is redundant given the checklist below, but I wonder if the high-level summary would still be useful.

```suggestion
Please link to an open-source repository and release that should be used for review. It does not suffice to point at an already built plugin: Mattermost code reviews and builds all plugins itself when listing in the marketplace.
```